lib/maintenance_tasks/engine.rb

```diff
@@ -11,6 +11,10 @@ class Engine < ::Rails::Engine
       eager_load! unless Rails.autoloaders.zeitwerk_enabled?
     end
 
+    initializer "maintenance_tasks.configs" do
+      MaintenanceTasks.backtrace_cleaner = Rails.backtrace_cleaner
```

Comment:
```suggestion
      MaintenanceTasks.backtrace_cleaner ||= Rails.backtrace_cleaner
```

You could move configuration to a rails config option (eg. `Rails.application.config.maintenance_tasks.*`) in the engine, but this should work fine. Given there's already config options that don't follow this pattern, I think it is fine as-is.